javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  const noStrings = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This will only save data that is a `number` data type, and we want everything that is not a `string`